public/app/features/admin/ServerStats.tsx

```diff
@@ -81,14 +85,27 @@ export const ServerStats = () => {
             content={[
               { name: 'Organisations', value: stats?.orgs },
               { name: 'Users total', value: stats?.users },
-              { name: 'Active users in last 30 days', value: stats?.activeUsers },
               ...(config.featureToggles.displayAnonymousStats && stats?.activeDevices
+                ? [{ name: 'Active users in last 30 days', value: activeDevicesAndUsers }]
```

Comment:
again, now this is getting complicated, can we construct this content array outside of the jsx, like just before the return statement?

that will let you avoid the ternaries and just construct the base array then do 
```tsx
if (<condition>) {
  <baseArray>.push(<something>);
}
```

Reply:
added 

```js
          <ServerStatsCard
            isLoading={isLoading}
            content={[
              { name: 'Organisations', value: stats?.orgs },
              { name: 'Users total', value: stats?.users },
              { name: 'Active users in last 30 days', value: stats?.activeUsers },
              ...getAnonymousStatsContent(stats, config),
              { name: 'Active sessions', value: stats?.activeSessions, indent: true },
            ]}
```